### PR TITLE
Jans auth server force request object

### PR DESCRIPTION
**Pull request by qbert2k**
### Prepare

- [ ] Read contribution guidelines
- [ ] Read license information

-------------------

### Description

- Target issue #378
  

- Implementation Details
  
    - new configuration entry:
    ``` 
    {
        ...
        "forceSignedRequestObject": true,
        ...
    }
    ```

-------------------
### Document the changes

- [ ] Relevant documentation has been updated if any (i.e. user guides, installation and configuration guides, technical design docs etc)

### CODE REVIEW

File: automation/rancher-partner-charts/app-readme.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## Introduction
```
Comment (yuriyz) [inline automation/rancher-partner-charts/app-readme.md:1]: @qbert2k it seems to not belong to this PR. Changes in `automation/rancher-partner-charts`.

**Review by yuriyz — CHANGES_REQUESTED** [review]
PR looks good but it seems to contain commits (in `/automation`) which does not belong to this PR, please check.

**Review by moabu — CHANGES_REQUESTED** [review]
Please close and reopen a new PR . This looks contaminated because you didn't pull in the changes before you created the branch .
